src/main/java/org/folio/circulation/domain/anonymization/checkers/FeesAndFinesClosePeriodChecker.java

```diff
@@ -27,7 +29,7 @@ public boolean canBeAnonymized(Loan loan) {
       .map(Account::getClosedDate)
       .filter(Optional::isPresent)
       .map(Optional::get)
-      .max(DateTime::compareTo);
+      .max((i, j) -> compareToMillis(i, j));
```

Comment:
That is the DateTime::compareTo, which confirms to milliseconds.
When I start Stage 7, ZonedDateTime will have a different comparison granularity, causing things to fail.

This is done purely to allow for a safe transition from DateTime into ZonedDateTime.

Reply:
Thank you for the explanation.

I wasn't asking about the reason for using a different method, I was asking why a method reference couldn't be used.

I just checked and it can be.

Apologies for the confusion.